sdk/src/math/orders.ts

```diff
@@ -160,7 +160,10 @@ export function getLimitPrice(
 	if (hasAuctionPrice(order, slot)) {
 		limitPrice = getAuctionPrice(order, slot, oraclePriceData.price);
 	} else if (order.oraclePriceOffset !== 0) {
-		limitPrice = oraclePriceData.price.add(new BN(order.oraclePriceOffset));
+		limitPrice = BN.max(
+			oraclePriceData.price.add(new BN(order.oraclePriceOffset)),
+			ONE
```

Comment:
this is alternative to actually grabbing tick size?

does it get corrected downstream?